clients/drcachesim/simulator/cache_simulator.cpp

```diff
@@ -440,9 +440,6 @@ cache_simulator_t::process_memref(const memref_t &memref)
         return true;
     }
 
-    // We use a static scheduling of threads to cores, as it is
```

Comment:
Stale comment? Since we do track `trace2core_`.

Reply:
Yes stale since we do have cpu markers.